runtime/src/launch/slurm-gasnetrun_ibv/launch-slurm-gasnetrun_ibv.c

```diff
@@ -305,10 +305,7 @@ static char* chpl_launch_create_command(int argc, char* argv[],
 static void chpl_launch_cleanup(void) {
   if (!debug) {
     char command[2*FILENAME_MAX];
-    if (getenv("CHPL_LAUNCHER_USE_SBATCH") == NULL) {
-      sprintf(command, "rm %s", expectFilename);
-      system(command);
-    } else {
+    if (getenv("CHPL_LAUNCHER_USE_SBATCH") != NULL) {
       sprintf(command, "rm %s", slurmFilename);
       system(command);
```

Comment:
This isn't your doing, but this seems fragile.  Can we use `remove(slurmFilename)` instead?

Reply:
We use `unlink(slurmFilename)` in the slurm-srun launcher. Ok with switching to that or is there some reason to prefer `remove()`?

Reply:
That's fine.  It's probably best to do whichever we do elsewhere.  `remove()` is from the C standard and `unlink()` is from the POSIX standard.